streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedKStreamImpl.java

```diff
@@ -89,6 +92,20 @@
         return aggregate(initializer, Materialized.with(keySerde, null));
     }
 
+    @Override
+    public <W extends Window> TimeWindowedCogroupedKStream<K, VOut> windowedBy(final Windows<W> windows) {
+        Objects.requireNonNull(windows, "windows can't be null");
+        return new TimeWindowedCogroupedKStreamImpl<>(windows,
+                builder,
+                sourceNodes,
+                name,
+                keySerde,
+                null,
```

Comment:
If there any case for which we would pass a non-null valueSerde here? If not, we should remove the parameter entirely.